runtime/integration-tests/src/cases/liquidity_pools.rs

```diff
@@ -156,9 +156,7 @@ mod utils {
 							token_name: BoundedVec::<
 								u8,
 								<T as pallet_pool_system::Config>::StringLimit,
-							>::try_from(
-								"A highly advanced tranche".as_bytes().to_vec()
-							)
+							>::try_from("A highly advanced tranche".as_bytes().to_vec())
```

Comment:
I think this breaks `cargo fmt`?

Reply:
Yeah, already fixed. I don't know why but my IDE Rustrover keeps introducing this fmt error and I can only commit the fixed version manually without the IDE.... Small indie company JetBrains..

Reply:
Could it be done regarding the cargo version? 

For me, what's works as CI is `cargo +nightly fmt` which is the action I've configured in my IDE